### ISSUE TITLE
Better structure for countries with multiple gov't levels

### ISSUE THREAD

**Issue by hejjoe** [govdirectory/website#212]
### The feature

The government regions of Baden-Württemberg should be a clearly visible subcategory of the German state Baden-Württemberg. The currently listed Regierungsbezirke should be a subcategory of the state of Hessen respectively the state of Bavaria. A similar structure could apply to other.

### Why?

E.g. in Germany there are three levels (federal level, state level, municipal level). So it could be useful to make the more influential federal level more visible than the lower levels.

### Examples and inspiration

So also https://de.wikipedia.org/wiki/Datei:Administrative_divisions_of_Germany.svg

**Comment by Ainali** [comment]
Thanks for creating the issue! We did talk about this a bit in our collab hour two weeks ago. Other similar countries would be USA, Russia and India and probably a bunch more.

We don't yet have a clear idea of how to do this best on the website. Should it be better filtering options, tabs on the country page or subpages another level down below country level?

So while we recognize the need for this, we need to work a bit on the solution.

**Comment by hejjoe** [comment]
Maybe using functions like expand/collapse could be a good website design idea? https://accessibility.huit.harvard.edu/technique-expandable-sections

**Comment by Ainali** [comment]
Yes, that is also a possibility. But I think we need to make a few sketches or experiments here to see how it really plays out. It would also be easier to experiment with if we had more data in Wikidata where this was modeled well. That might be the first task to get done.

**Comment by Ainali** [comment]
Here is a very rough test. It inserts a new heading with details / summary under the header, but before the agencies. When clicked on, it reveals a list similar to the one on the countries page. And then each entity on a lower level get their own page as a subpage under the country, in this case for example `russia/adygea/`. Would that work? The tricky part is perhaps to name the unfoldable header right. I am not sure if we can get it easily from Wikidata, and if we use too generic a name, people might not understand it since they would expect something specific.

![Skärmbild från 2022-11-26 21-15-12](https://user-images.githubusercontent.com/5566159/204107567-7b0583da-3672-44d7-bf94-d875e8833d89.png)

![Skärmbild från 2022-11-26 21-15-19](https://user-images.githubusercontent.com/5566159/204107629-b7828a47-b363-4d1c-bb19-6979dfbb1b1a.png)

**Comment by Abbe98** [comment]
Heads up! We will start doing some testing by moving Scottish agencies from the current [United Kingdom](https://www.govdirectory.org/united-kingdom/) page into a subpage.

**Comment by RVA2869** [comment]
Are there any updates? Because that's what I'm waiting for. So that I can add the rest of Belgium.

**Comment by Abbe98** [comment]
> Are there any updates? Because that's what I'm waiting for. So that I can add the rest of Belgium.

I'm still experimenting I'm afraid(about to publish a work in progress branch/merge request), a big drawback for no is that it increases the complexity of "add a country" quite a lot.
